SignalServiceKit/src/Messages/DeviceSyncing/OWSOutgoingSentMessageTranscript.m

```diff
@@ -48,15 +49,21 @@ - (OWSSignalServiceProtosSyncMessageBuilder *)syncMessageBuilder
 
     OWSSignalServiceProtosSyncMessageSentBuilder *sentBuilder = [OWSSignalServiceProtosSyncMessageSentBuilder new];
     [sentBuilder setTimestamp:self.message.timestamp];
-    [sentBuilder setDestination:self.message.recipientIdentifier];
-    [sentBuilder setMessage:[self.message buildDataMessage:self.message.recipientIdentifier]];
+    [sentBuilder setDestination:self.recipientIdentifierForMessage];
```

Comment:
Should we just hardcode `TSAccountManager.localNumber` rather than opening a sneaky transaction here?

Reply:
Good idea.